Lib/fontTools/varLib/designspace.py

```diff
@@ -66,11 +77,14 @@ def _load(et):
 	for et in ds.find('instances'):
 		instances.append(_loadItem(et))
 
-	return masters, instances, axisMap
+	return axes, masters, instances
 
 def load(filename):
-	"""Load designspace from a file name or object.  Returns two items:
-	list of masters (aka sources) and list of instances."""
+	"""Load designspace from a file name or object.
+	   Returns three items:
```

Comment:
Thanks Miguel. Talking with @behdad yesterday, we were thinking of having the load and _load function return a dictionary instead of a tuple of two, now three (and tomorrow who knows how many) items. Could you also  make that change here?